dispatch/static/manager/src/js/components/ZoneEditor/WidgetField.js

```diff
@@ -7,14 +7,29 @@ import { FormInput } from '../inputs'
 export default function WidgetField(props) {
   const Field = fields[props.field.type]
 
+  let fieldError = ''
+  let childErrors = null
+  if (props.field.type == 'widget' && props.error) {
+    try {
+      childErrors = JSON.parse(props.error)
+      fieldError = childErrors.self
+    } catch (e) {
+      fieldError = props.error
+    }
+  } else {
+    fieldError = props.error
+  }
+
   return (
     <FormInput
-      error={props.error}
+      error={fieldError}
       label={props.field.label}>
       <Field
+        errors={childErrors}
         label={props.field.label}
         many={props.field.many}
         data={props.data}
+        cWidgets={props.field.widgets}
```

Comment:
Maybe instead of passing `label`, `many` and `cWidgets`, we could just pass the entire `field` object?